tests/main/special-home-can-run-classic-snaps/task.yaml

```diff
@@ -0,0 +1,30 @@
+summary: Check that users with homes in /var/lib can run *classic* snaps
```

Comment:
There seems to be some overlap with `tests/main/special-home`. Would it make sense to squash both tests into one?

Reply:
Maybe, I'm sitting on the fence about it. The jenkins user is really the only user with a non /home  HOME dir that can run confined snaps at this point. But all things that use /var/lib/ as home should be able to run classic snaps again. It's tricky, we are still regressing on confined snaps but to fix that we would have to revert the quirks change entirely (which we can do, we have two closed PRs that do that which we just need to reopen).